code/modules/mob/living/carbon/human/species_types/plasmamen.dm

```diff
@@ -162,6 +162,15 @@
 			to_chat(H, "<span class='warning'>The excess milk is dripping off your bones!</span>")
 		H.heal_bodypart_damage(1.5,0, 0)
 		H.reagents.remove_reagent(chem.type, chem.metabolization_rate)
+		for(var/i in H.all_wounds)
+			var/datum/wound/W = i
+			W.on_xadone(2)
+		return TRUE
+	if(chem.type == /datum/reagent/toxin/plasma)
+		H.reagents.remove_reagent(chem.type, chem.metabolization_rate)
+		for(var/i in H.all_wounds)
+			var/datum/wound/W = i
+			W.on_xadone(4) // plasmamen use plasma to reform their bones or whatever

```

Comment:
but not stabilized plasma, apparently

still, this is nice, thanks for this

Reply:
will the livers of plasmamen still attempt to purge plasma from them, due to it technically being a toxin?